packages/lambda-tiler/src/import/make.cog.ts

```diff
@@ -0,0 +1,29 @@
+import { JobCreationContext } from '@basemaps/cli/build/cog/cog.stac.job';
+import { TileMatrixSet } from '@basemaps/geo';
+import { Env } from '@basemaps/shared';
+import { RoleConfig } from './imagery.find.js';
+
+export async function getJobCreationContext(
+  path: string,
+  tileMatrix: TileMatrixSet,
+  role: RoleConfig,
+  files: string[],
+): Promise<JobCreationContext> {
+  const bucket = Env.get(Env.ImportImageryBucket);
+  if (bucket == null) throw new Error('Output AWS s3 bucket Not Found.');
+  const ctx: JobCreationContext = {
+    override: {
+      projection: tileMatrix.projection,
+      resampling: {
+        warp: 'bilinear',
+        overview: 'lanczos',
+      },
+    },
+    outputLocation: { type: 's3' as const, path: `s3://${bucket}` },
```

Comment:
does this path need to be more specific? `s3://${bucket}/${ulid}` ?

Reply:
Yes, we can't add id here until we got the context first. So I add it in the `import()` function.
`// Add id back to JobCreationContext
    ctx.override!.id = id;
    ctx.outputLocation.path = fsa.join(ctx.outputLocation.path, id);`